.htmllintrc

```diff
@@ -3,5 +3,6 @@
   "head-req-title": null,
   "id-class-style": "camel",
   "indent-width": 2,
-  "indent-width-cont": true
+  "indent-width-cont": true,
+  "img-req-alt": false
```

Comment:
Per https://github.com/htmllint/htmllint/wiki/Options#img-req-alt, lets change this to:

```json
"img-req-alt": "allownull"
```

That should allow an empty string, and still throw errors if we have an `<img>` tag without an `alt=` attribute.

Reply:
And you're ok with that?

Reply:
Ah, ok. I'll do that and merge the final result. Thanks!